packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -651,12 +651,9 @@ class RCTAztecView: Aztec.TextView {
     private func applyFontConstraints(to baseFont: UIFont) -> UIFont {
         let oldDescriptor = baseFont.fontDescriptor
         let newFontSize: CGFloat
+        let fontMetrics = UIFontMetrics(forTextStyle: .body)
 
-        if let fontSize = fontSize {
-            newFontSize = fontSize
-        } else {
-            newFontSize = baseFont.pointSize
-        }
+        newFontSize = fontMetrics.scaledValue(for: fontSize ?? baseFont.pointSize)
```

Comment:
This makes sense to me @SiobhyB!

One optional formatting nit. Since this is now a single line we could update it to:

```swift
let newFontSize = fontMetrics.scaledValue(for: fontSize ?? baseFont.pointSize)
```

Reply:
Thank you, Tanner! I updated in https://github.com/WordPress/gutenberg/pull/57339/commits/d98f0b762c0446ae92bac005d9c650352e79e829.